src/router/router.js

```diff
@@ -1,64 +1,50 @@
-import VueRouter from 'vue-router';
 import Vue from 'vue';
+import VueRouter from 'vue-router';
 import { routeParser } from 'vue-i18n-manager';
+import localeConfig from '../config/localeConfig';
+import configManager from '../util/configManager';
 import { PropertyNames, VariableNames } from '../data/enum/configNames';
-import getLocaleConfig from '../config/localeConfig';
-import { CONFIG_MANAGER } from '../data/Injectables';
-import { getValue } from '../util/injector';
-
 import routes from './routes';
 
 Vue.use(VueRouter);
 
-let router = null;
-
-const getRouter = () => {
```

Comment:
Are you aware that you are changing the behaviour of the running code with this change(s)?
Previously, the code in this function would only be executed when first called. Now it's executed when required (first thing). So will loose some control.

Reply:
Let's say the router setup needs a value that is set in another file. You would like to control the order of how things are executed. Having to rely on changing the import order to (hopefully) change the order of execution is error-prone and unclear.
Explicitly calling functions to specify the order of execution gives you more control.